### PR TITLE
Changes in class

### PR DESCRIPTION
**Pull request by Jaikumar1998**
review the changes

### CODE REVIEW

File: model/src/test/java/model/model1.java [hunk 1]
```diff
@@ -1,6 +1,11 @@
 package model;
 
 public class model1 {
+	
+	public void model() {
+		String name = "jai";
```
Comment (pranitamandal) [inline model/src/test/java/model/model1.java:6]: Change the string name
Do not hard code the data

**Review by pranitamandal — COMMENTED** [review]
added few comments

File: model/src/test/java/model/model1.java (hunk 1 above)
Comment (Jaikumar1998) [reply model/src/test/java/model/model1.java:6]: fixed

**Comment by pranitamandal** [comment]
merged

**Pull request merged**